dashboard.go

```diff
@@ -7,19 +7,23 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"net/url"
 	"os"
 )
 
 type DashboardMeta struct {
-	IsStarred bool   `json:"isStarred"`
-	Slug      string `json:"slug"`
-	Folder    int64  `json:"folderId"`
+	IsStarred   bool   `json:"isStarred"`
+	Slug        string `json:"slug"`
+	Folder      int64  `json:"folderId"`
+	FolderTitle string `json:"folderTitle"`
 }
 
+// DashboardSaveResponse grafana response for create dashboard
 type DashboardSaveResponse struct {
 	Slug    string `json:"slug"`
-	Id      int64  `json:"id"`
-	Uid     string `json:"uid"`
+	ID      int64  `json:"id"`
+	UID     string `json:"uid"`
```

Comment:
I don't think it's a great idea to be renaming these, as we tend to follow the `Id` and `Uid` convention for these fields. Would you be alright keeping these as-is?